packages/api-key-service/package.json

```diff
@@ -3,24 +3,29 @@
   "dependencies": {
     "@aws/dynamodb-data-mapper-annotations": "^0.7.3",
     "@homeservenow/serverless-aws-handler": "^0.2.4",
-    "@reapit/foundations-ts-definitions": "workspace:packages/foundations-ts-definitions",
     "class-transformer": "^0.4.0",
     "class-validator": "^0.13.1"
   },
   "devDependencies": {
     "@reapit/config-manager": "workspace:packages/config-manager",
+    "@reapit/foundations-ts-definitions": "workspace:packages/foundations-ts-definitions",
+    "@reapit/ts-scripts": "workspace:packages/ts-scripts",
+    "@reapit/utils": "workspace:packages/utils",
     "concurrently": "^6.2.0",
+    "cross-env": "^7.0.3",
     "eslint": "^7.27.0",
+    "jest": "26.6.3",
     "prettier": "^2.3.0",
     "serverless": "^2.43.1"
   },
   "scripts": {
-    "start": "serverless dynamodb install && yarn build && yarn serverless offline start --runtime nodejs12.x --stage local",
+    "start": "yarn publicKeyFetch && yarn build && yarn serverless offline start --runtime nodejs12.x --stage local",
```

Comment:
Can you not just do `yarn reapit-fetch-public-keys` - should be accessible to yarn as you have `@reapit/utils` in scope. Small point so don't worry about it

Reply:
yea might as well 👍